packages/components/src/Loading/LoadingSpinner/LoadingSpinner.tsx

```diff
@@ -2,16 +2,98 @@ import React, { HTMLAttributes } from "react"
 import classnames from "classnames"
 import { VisuallyHidden } from "~components/VisuallyHidden"
 import { OverrideClassName } from "~components/types/OverrideClassName"
-import styles from "./LoadingSpinner.module.scss"
+import styles from "./LoadingSpinner.module.css"
+
+type LoadingSpinnerSize = "xs" | "sm" | "md"
```

Comment:
Anyone think this is worth exporting?

Reply:
I can't see anyone wanting this so yeah I'd leave it private unless/until someone asks for it

Reply:
Wouldn't even make a type for it tbh 😅